lib/tasks/elasticsearch.rake

```diff
@@ -1,32 +0,0 @@
-require "elasticsearch/rails/tasks/import"
```

Comment:
In the development setup docs, we ask people to call `bundle exec rake environment elasticsearch:import:all DIR=app/models FORCE=y` to import models. Should update the docs and ask people to call something else now (something that calls `Rubygem.searchkick_reindex`?)

Reply:
updated